### PR TITLE
Bug 1881820: Fix external mode queries for RGW Performance Card

### PR DESCRIPTION
**Pull request by bipuladh**
- Fix the queries for External Mode
- Show an empty graph for empty data frames. 
- Add dependency array for Infrastructure Detection Logic in Ceph
After Fix:
Screenshot from 2020-09-23 19-57-14

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
           setIndepModeSupportedPlatform(true);
         }
       });
-  });
+  }, []);
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx:79]: Do we need useEffect if this is not going refreshed base on some parameters?

**Comment by anmolsachan** [comment]
metrics look good to me

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx (hunk 1 above)
Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx:79]: Discussed in DMs

**Comment by cloudbehl** [comment]
/lgtm

nit: We might need to rethink calling `bandwidth` as `Throughput`.

**Pull request merged**